testem.json

```diff
@@ -3,8 +3,15 @@
   "cwd": ".dist-test",
   "test_page": "index.html?hidepassed",
   "disable_watching": true,
+  "launchers": {
+    "NodeQUnit": {
+      "command": "node scripts/util/test-in-node.js",
```

Comment:
Scripts that are run directly shouldn't live in util. They should be runnable scripts.

Util is for things that are required by other scripts.